docs/02.API-REFERENCE.md

```diff
@@ -1084,6 +1084,84 @@ jerry_get_global_object (void);
 
 Functions to check the type of an API value ([jerry_value_t](#jerry_value_t)).
 
+
+## jerry_value_instanceof
+
+**Summary**
+
+Returns whether the given `jerry_value_t` is an instance of the given prototype.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_value_instanceof (const jerry_value_t value, const jerry_value_t prototype);
+```
+
+- `value` - api value of instance object
+- `prototype` - api value of prototype
+- return value
+  - error, if value argument is not an object or prototype argument is not a function
```

Comment:
There should be a test where the input `value` (and also for the `prototype`) is not an object to trigger this case.